packages/base-dao/tools/__snapshots__/generateDaoSchema-test.js.snap

```diff
@@ -57,19 +57,19 @@ export default class SurveyDAO {
     return this.entityDAO.surveysConnection(parent, args, context);
   }
 
-  getRelatedEntityId: async (originalEntityId: string, relationEntityName: string) => {
+  async getRelatedEntityId(originalEntityId: string, relationEntityName: string): Promise<string | null> {
     return this.entityDAO.getRelatedEntityId(originalEntityId, relationEntityName);
```

Comment:
Since the base dao type is any, it won't allow passing the generics to the underlying base dao